### PR TITLE
[ZEPPELIN-1208] fixed invaild position user config button in navbar

### PR DESCRIPTION
**Pull request by cloverhearts**
### What is this PR for?
icon for connect or disconnect
invalid align in the mobile screen.
See the screen shot.

### What type of PR is it?
Bug Fix

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1208

### How should this be tested?
Loading a web page from a mobile or Minimizing the width web browser.

#### before
cap 2016-07-19 01-22-56-717
cap 2016-07-19 01-23-08-542

#### after
cap 2016-07-19 01-21-22-964
cap 2016-07-19 01-21-32-983

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-web/src/components/navbar/navbar.html [hunk 1]
```diff
@@ -95,6 +93,9 @@
             </ul>
           </div>
         </li>
+        <li>
```
Comment (corneadoug) [inline zeppelin-web/src/components/navbar/navbar.html]: @cloverhearts Can you remove this empty <li></li>?

Comment (cloverhearts) [inline zeppelin-web/src/components/navbar/navbar.html]: @corneadoug 
thank you.
I have removed the empty li tags.

**Comment by corneadoug** [comment]
LGTM

**Comment by corneadoug** [comment]
Merging if there is no more discussions

**Pull request closed without merging**